ThirdPartyNotices.txt

```diff
@@ -5,9 +5,10 @@ Do Not Translate or Localize
 
 This project incorporates components from the projects listed below. The original copyright notices and the licenses under which the .NET Foundation received such components are set forth below. The .NET Foundation reserves all rights not expressly granted herein, whether by implication, estoppel or otherwise.
 
-1.	PedroLamas/DeferredEvents version 1.0.4 (https://github.com/PedroLamas/DeferredEvents), included in Microsoft.Toolkit.Uwp/Deferred.
+1.	PedroLamas/DeferredEvents version 1.0.4 (https://github.com/PedroLamas/DeferredEvents).
```

Comment:
Bad merge for this line?

Reply:
Whoops, good eye! Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3229/commits/d5fbe0531447f87e2638b781a3902b4df586677b.